### DISCUSSION TITLE
revalidate tag without fetch

### DISCUSSION THREAD

**Discussion by HassanIyan** [vercel/next.js#56691] [App Router]
```javascript
export default async function Home() {
    const posts = await prisma.post.findMany();
    return (
        <main className='min-h-screen w-screen'>

        </main>
    )
}
``` 
for this code is there a way to give a tag and revalidate the tag without using fetch.

**Comment by SidKH** [comment]
If you want to get fresh data on every page load, you can make the page dynamic.

```jsx
export const dynamic = 'force-dynamic'

export default async function Home() {
    const posts = await prisma.post.findMany();
    return (
        <main className='min-h-screen w-screen'>

        </main>
    )
}
```

If you want to keep data cached, and revalidate it on demand, check out https://nextjs.org/docs/app/building-your-application/caching#unstable_cache
